src/env.h

```diff
@@ -318,7 +320,7 @@ constexpr size_t kFsStatsBufferLength = kFsStatsFieldsNumber * 2;
   V(write_host_object_string, "_writeHostObject")                              \
   V(write_queue_size_string, "writeQueueSize")                                 \
   V(x_forwarded_string, "x-forwarded-for")                                     \
-  V(zero_return_string, "ZERO_RETURN")
+  V(zero_return_string, "ZERO_RETURN")                                         \
```

Comment:
I don't think we have a standard practice of dangling backslashes in macros do we? at least the next one doesn't do this.

Reply:
I guess I was scanning from the top, the previous two define-lists have the dangling backslash, but its irrelevant to this PR, I removed.